supabase/migrations/20230908133828_add-feedback-table.sql

```diff
@@ -0,0 +1,46 @@
+create table "public"."feedback" (
+    "id" bigint generated by default as identity not null,
+    "created_at" timestamp with time zone default now(),
+    "feedback_positive" boolean,
+    "conversation_id" bigint not null,
+    "system_prompt" text not null,
+    "user_message_idx" bigint
```

Comment:
So is this the index of the most recent user message after which the user gave feedback?

What if we were to change how we decide when to ask for feedback in the future?

Is there something simpler we could store to identify when in the conversation that feedback was given?

Reply:
how about this?